WineCellar/WineCellar.GUI/appsettings.json

```diff
@@ -1,5 +1,5 @@
 ﻿{
   "ConnectionStrings": {
-    "DefaultConnection": ""
+    "DefaultConnection": "Data Source=127.0.0.1;Initial Catalog=WineDB;User ID=sa;Password=WineCellar21!;Connect Timeout=60;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False"
```

Comment:
Dit lijkt me een foutje, misschien moeten we even een gitignore maken voor de appsettings?